README.md

```diff
@@ -1,2 +1,2 @@
 # Kubernetes Tests
-CI for Kubernetes Tests
+CI for SRIOV CNI
```

Comment:
It would be better to write 
CI for deploying Kubernetes with Multus, SRIOV CNI, and Device Plugin